conjure-go-client/httpclient/client_params.go

```diff
@@ -183,6 +183,15 @@ func WithMetrics(tagProviders ...TagsProvider) ClientOrHTTPClientParam {
 	})
 }
 
+// WithoutMetrics disables the "client.response" metric.
+func WithoutMetrics(tagProviders ...TagsProvider) ClientOrHTTPClientParam {
```

Comment:
The `tagProviders` parameter is unused and should be removed

Reply:
Done